test/jasmine/tests/gl2d_click_test.js

```diff
@@ -346,6 +346,36 @@ describe('Test hover and click interactions', function() {
         .then(done);
     });
 
+    it('@gl should not error when scattergl trace has missing points', function(done) {
```

Comment:
Interestingly when passing a blank string instead of null there is no error. 
Something like ``` x: [1, 2, 3, 4], y: [10, 15, 16, '' ]```
That may considered to be useful if one wants to set the last items of array to undefined values (without losing the length). 
But curious to know where that handled?

Reply:
Hmm. I'm getting `Cannot read property 't' of undefined` errors on https://codepen.io/etpinard/pen/LYPqWbb which has `''` swapped in for `null`